docs/operating-scylla/procedures/backup-restore/backup.rst

```diff
@@ -29,11 +29,16 @@ With time, SSTables are compacted, but the hard link keeps a copy of each file.
 
 | 1. Data can only be restored from a snapshot of the table schema, where data exists in a backup. Backup your schema with the following command:
 
-| ``$: cqlsh -e "DESC SCHEMA" > <schema_name.cql>``
+| ``$: cqlsh -e "DESC SCHEMA WITH INTERNALS" > <schema_name.cql>``
 
 For example:
 
-| ``$: cqlsh -e "DESC SCHEMA" > db_schema.cql``
+| ``$: cqlsh -e "DESC SCHEMA WITH INTERNALS" > db_schema.cql``
+
+.. warning::
+
+  To get proper schema description, you need to use cqlsh at least in version ``16.0.19``. Using older version of cqlsh to
+  generate a schema backup may lead to data resurrection or data loss. To check version of your cqlsh, you can use ``cqlsh --version``.
```

Comment:
> Using older version of cqlsh to generate a schema backup may lead to data resurrection or data loss.

It's not exactly true. Creating a schema backup with an older version of cqlsh will not lead to data resurrection or loss. Restoring such a backup might lead to it.

Reply:
I suggest to change this into the following:

```
Restoring a schema backup created by an older version of cqlsh may lead to data resurrection or data loss.
```

@annastuchlik does this sound good to you?